scripts/windows/make.ps1

```diff
@@ -145,28 +135,29 @@ function bundle (
 function lint() {
     compile_deps
     golint -set_exit_status ./cmd/... ./internal/...
-    if ($lastexitcode -ne 0){ throw }
 }
 
 function vendor() {
     go mod tidy
-    if ($lastexitcode -ne 0){ throw }
     go mod vendor
-    if ($lastexitcode -ne 0){ throw }
 }
 
 function vet() {
     compile_deps
+    $ErrorActionPreference = "Continue"
     go vet -mod vendor ./... 2>&1 | Select-String -Pattern "\.go" | Select-String -NotMatch -Pattern "_test\.go" -outvariable gofiles
```

Comment:
Can you use `-ErrorAction Ignore` for the command instead of having to change the global flag?

Reply:
> Can you use `-ErrorAction Ignore` for the command instead of having to change the global flag?

Tried that, but unfortunately it looks like `ErrorAction` can't override `$ErrorActionPreference = Stop`.  See https://github.com/PowerShell/PowerShell/issues/7623#issuecomment-415581590.

Reply:
Ah, that's too bad. I guess could also wrap in try/catch? 🤷‍♂️ I'm fine with this though.